cpp/benchmarks/fixture/nvbench_main.cu

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Just curious, will this work, if this `nvbench_main.cu` is renamed to `nvbench_main.cpp`?

(Since compiling with g++ is faster than nvcc).

Reply:
Yes, that works. Updated.
Thanks.